pkgs/by-name/au/authentik/package.nix

```diff
@@ -0,0 +1,226 @@
+
+    postPatch = "cd website";
+
+    installPhase = "cp -r help $out";
+
+    npmInstallFlags = [ "--include=dev" ];
+    npmBuildScript = "build-docs-only";
+  };
+
+  clientapi = stdenvNoCC.mkDerivation {
+    pname = "authentik-client-api";
+    inherit version src;
+
+    patchPhase = ''
+      rm Makefile
+
+      substituteInPlace ./scripts/api-ts-config.yaml \
+        --replace '/local' "$(pwd)/"
+    '';
+
+    nativeBuildInputs = [ openapi-generator-cli ];
+    buildPhase = ''
+      openapi-generator-cli generate -i ./schema.yml \
+      -g typescript-fetch -o $out \
+      -c ./scripts/api-ts-config.yaml \
+      --additional-properties=npmVersion=${nodePackages.npm.version} \
+      --git-repo-id authentik --git-user-id goauthentik
+    '';
+  };
+
+  webui = buildNpmPackage {
+    pname = "authentik-webui";
+    inherit version;
+    src = runCommand "authentik-webui-source" {} ''
+      mkdir -p $out/web/node_modules/@goauthentik/
+      cp -r ${src}/web $out/
+      cp -r ${src}/website $out/
+      cp -r ${clientapi} $out/web/node_modules/@goauthentik/api
+    '';
+    npmDepsHash = "sha256-5aCKlArtoEijGqeYiY3zoV0Qo7/Xt5hSXbmy2uYZpok=";
+
+    postPatch = "cd web";
+
+    installPhase = ''
+      mkdir $out
+      cp -r dist $out/dist
+      cp -r authentik $out/authentik
+    '';
+
+    NODE_ENV = "production";
+    NODE_OPTIONS = "--openssl-legacy-provider";
+
+    npmInstallFlags = [ "--include=dev" ];
+  };
+
+  authentik-django = python3.pkgs.buildPythonPackage {
+    pname = "authentik-django";
+    inherit version src;
+    format = "pyproject";
+
+    patchPhase = ''
+      substituteInPlace authentik/root/settings.py \
+        --replace 'Path(__file__).absolute().parent.parent.parent' "\"$out\""
+      substituteInPlace authentik/lib/default.yml \
+        --replace '/blueprints' "$out/blueprints"
+    '';
+
+    nativeBuildInputs = [ python3.pkgs.poetry-core ];
+    propagatedBuildInputs = with python3.pkgs; [
+      argon2-cffi
+      celery
+      channels
+      channels-redis
+      colorama
+      dacite
+      daphne
+      deepmerge
+      defusedxml
+      django
+      django-filter
+      django-guardian
+      django-model-utils
+      django-prometheus
+      django-redis
+      djangorestframework
+      djangorestframework-guardian2
+      docker
+      drf-spectacular
+      duo-client
+      facebook-sdk
+      flower
+      geoip2
+      gunicorn
+      httptools
+      kubernetes
+      ldap3
+      lxml
+      opencontainers
+      packaging
+      paramiko
+      psycopg
+      pycryptodome
+      pydantic
+      pydantic-scim
+      pyjwt
+      pyyaml
+      requests-oauthlib
+      sentry-sdk
+      structlog
+      swagger-spec-validator
+      twilio
+      twisted
+      ua-parser
+      urllib3
+      uvicorn
+      uvloop
+      watchdog
+      webauthn
+      websockets
+      wsproto
+      xmlsec
+      zxcvbn
+      jsonpatch
+    ] ++ [
```

Comment:
```suggestion
```

Reply:
This is necessary because codespell is also in `python3.pkgs` but throws an error saying it moved to the toplevel